src/__tests__/RSTests/Card/card_1.test.tsx

```diff
@@ -0,0 +1,328 @@
+        effect: 'Antiseptic, cleaned wounds',
+        image:
+          'https://static.wikia.nocookie.net/harrypotter/images/a/a0/WoundCleaningPotionHM.png',
+        inventors: null,
+        ingredients: null,
+        manufacturers: null,
+        name: 'Wound-Cleaning Potion',
+        side_effects: null,
+        time: null,
+        wiki: 'https://harrypotter.fandom.com/wiki/Wound-Cleaning_Potion',
+      },
+      links: {
+        self: '/v1/potions/af751624-ca96-4cb3-b272-ad284f28664b',
+      },
+    },
+  ],
+  meta: {
+    pagination: {
+      current: 4,
+      first: 1,
+      prev: 3,
+      records: 96,
+    },
+    copyright: 'Copyright © Potter DB 2023',
+    generated_at: '2023-11-12T10:47:24.534+00:00',
+  },
+  links: {
+    self: 'https://api.potterdb.com/v1/potions?sort=name&page[size]=29&page[number]=4&filter[name_cont_any]=a',
+    current:
+      'https://api.potterdb.com/v1/potions?filter[name_cont_any]=a&page[number]=4&page[size]=29&sort=name',
+    first:
+      'https://api.potterdb.com/v1/potions?filter[name_cont_any]=a&page[number]=1&page[size]=29&sort=name',
+    prev: 'https://api.potterdb.com/v1/potions?filter[name_cont_any]=a&page[number]=3&page[size]=29&sort=name',
+  },
+};
+
+const mockCardResp = {
+  data: {
+    id: '69f97266-e62f-4505-aaed-d20762bd57ee',
+    type: 'potion',
+    attributes: {
+      slug: 'twilight-moonbeams',
+      characteristics: 'Red in colour',
+      difficulty: null,
+      effect: 'Causes the drinker to become infatuated with the giver of the potion',
+      image: 'https://static.wikia.nocookie.net/harrypotter/images/b/b7/TwilightMoonbeams.jpg',
+      inventors: 'Fred and George Weasley',
+      ingredients: 'Pearl Dust',
+      manufacturers: null,
+      name: 'Twilight Moonbeams',
+      side_effects: null,
+      time: null,
+      wiki: 'https://harrypotter.fandom.com/wiki/Twilight_Moonbeams',
+    },
+    links: {
+      self: '/v1/potions/69f97266-e62f-4505-aaed-d20762bd57ee',
+    },
+  },
+  meta: {
+    copyright: 'Copyright © Potter DB 2023',
+    generated_at: '2023-11-12T16:48:44.039+00:00',
+  },
+  links: {
+    self: 'https://api.potterdb.com/v1/potions/69f97266-e62f-4505-aaed-d20762bd57ee',
+  },
+};
+
+describe('Testing Card - should renders the relevant card data', async () => {
+  beforeAll(async () => {
+    vi.mock('../../../scripts/pages/App/loader', () => {
+      return {
+        loader: vi.fn(async (params: PotionsReqParams) => {
+          console.log(params);
+          const items = fakeRes.data;
+          const pagination = {
+            current: params.pagination.page,
+            pages: 1,
+          };
+          return { items, pagination };
+        }),
+      };
+    });
+
+    vi.spyOn(HpApi, 'getPotion').mockReturnValue(new Promise((resolve) => resolve(mockCardResp)));
+  });
+
+  afterAll(() => {
+    vi.clearAllMocks();
+    vi.resetAllMocks();
+  });
+
+  test('should renders valid ico', async () => {
```

Comment:
Not clear what do you mean by 'valid'. Consider to rename to `IconName icon rendered`